src/Alien.cpp

```diff
@@ -59,6 +56,7 @@ std::vector<std::string> Alien::GetShot() {
 void Alien::GetShot(const unsigned int timeout, std::vector<std::string>& bullets) {
    bullets.clear();
    if (!mBody) {
+      LOG(WARNING) << "Alien attempted to GetShot but is not properly initialized";
```

Comment:
It's definitely an unexpected condition as mBody is allocated in the constructor and a "CHECK" is done to make sure it is not null. I don't think the system can end up in this state; at least not in any normal sense.